kernel/packages/shared/ethereum/provider.ts

```diff
@@ -64,7 +64,7 @@ export function getProviderType() {
 }
 
 export async function awaitWeb3Approval(): Promise<void> {
-  if (!WORLD_EXPLORER) {
```

Comment:
@caminoezequiel how does this affect the preview mode?

Reply:
In preview mode without `ENABLE_WEB3` flag we are auto signing as GUEST. I changed it to prevent two authentications, but when builder is running we need to setup a `WebSocketProvider`